docs/sources/mimir/release-notes/v2.6.md

```diff
@@ -61,3 +61,4 @@ The following experimental options and features are now stable:
 - Querier: Canceled requests are no longer reported as "consistency check" failures. [PR 3837](https://github.com/grafana/mimir/pull/3837) [PR 3927](https://github.com/grafana/mimir/pull/3927)
 - Distributor: Don't panic when `metric_relabel_configs` in overrides contains null element. [PR 3868](https://github.com/grafana/mimir/pull/3868)
 - Ingester, Compactor: Fix panic that can occur when compaction fails. [PR 3955](https://github.com/grafana/mimir/pull/3955)
+- Security: Update Go to version 1.20.3 to fix CVE-2023-24538 [PR 4798](https://github.com/grafana/mimir/pull/4798)
```

Comment:
Doesn't this only apply to the patch release, and not v2.6 as a whole? (I read this file as describing v2.6)

Reply:
Yes, at first I thought this file would include bug fixes from patch releases, but looking at the last time there was a patch release, 2.3, only the changelog is updated and not the release notes. I'll delete this.